packages/next/src/server/config-schema.ts

```diff
@@ -200,6 +200,12 @@ const configSchema = {
       type: 'string',
       nullable: true,
     },
+    disableServerMinification: {
+      type: 'boolean',
+    },
+    enableSourceMapsForServer: {
```

Comment:
do we want to align the API meaning here like either use both enableXX or disableXX? I think it's easier for memorizing the configs, no need to tweak around the flag in the head